### DISCUSSION TITLE
May  join team now?

### DISCUSSION THREAD

**Discussion by IdkAnythin07** [ROM-builders/temporary#19779] [General]
Github: https://github.com/IdkAnythin07
Telegram: https://t.me/IdkAnythin07

DT: https://github.com/idkAnythin07/android_device_realme_RMX1941/tree/octavi
Local Manifest: https://github.com/idkAnythin07/local_manifest/tree/octavi

PR:
#19627 

Successful Builds:
https://cirrus-ci.com/github/ROM-builders/temporary/pull%2F19627

**Comment by shekhawat2** [comment]
No

**Reply by IdkAnythin07** [reply]
reason?

**Reply by shekhawat2** [reply]
because your account is inactive.

**Reply by IdkAnythin07** [reply]
?? what do you mean!?
i just build superior
and crdroid in queue

**Comment by IdkAnythin07** [comment]
@Apon77 sir?

**Comment by Apon77** [comment]
Please follow #learn instructions from the telegram group.

**Reply by IdkAnythin07** [reply]
i followed now the history of my dt for 'new trees' are better same with manifest
